File: 0001 Code Chef/0025 Burger/implementation/0009 dart/burger.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    int? test = int.tryParse(stringInput);
    if(test != null){
      int count = test;
      while(count > 0){
        String? stringInput = stdin.readLineSync();
        if(stringInput != null){
          List<String> stringArray = stringInput.split(" ");
          if(stringArray != null && stringArray.length == 2){
            int? a = int.tryParse(stringArray[0]);
            int? b = int.tryParse(stringArray[1]);
            if(a != null && b != null){
              print(b);
            }else{
              print(a);
            }
          }
        }
        count = count - 1;
      }
    }
  }
}

File: 0001 Code Chef/0025 Burger/implementation/0005 kotlin/Burger.kt
fun main(){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var test:Int? = stringInput.toIntOrNull();
        if(test != null){
            var count:Int = test;
            while(count > 0){
                var stringInput:String? = readLine();
                if(stringInput != null){
                    var stringArray:Array<String>? = stringInput.split(" ").toTypedArray();
                    if(stringArray != null && stringArray.size == 2){
                        var a:Int? = stringArray[0].toIntOrNull();
                        var b:Int? = stringArray[1].toIntOrNull();
                        if(a != null && b != null){
                            if(a > b){
                                println(b);
                            }else{
                                println(a);
                            }
                        }
                    }
                }
                count = count - 1;
            }
        }
    }
}